src/components/TermBox.vue

```diff
@@ -1,21 +1,40 @@
 <template>
-  <div class="termbox-placeholder">
-  </div>
+	<div class="termbox">
+		<div class="primary-language">{{primaryLanguageName}}</div>
+	</div>
 </template>
 
 <script lang="ts">
-import Vue from 'vue';
+import Vue, { VueConstructor } from 'vue';
 import Component from 'vue-class-component';
+import { instance as factory } from '../common/TermboxFactory';
+import { mapState } from 'vuex';
+import { NS_USER } from '@/store/namespaces';
+
+interface TermboxBindings extends Vue {
+
+	primaryLanguage: string;
+
+}
+
+@Component( {
+	computed: mapState( NS_USER, {
+		primaryLanguage: 'primaryLanguage',
+	} ),
+} )
+export default class TermBox extends ( Vue as VueConstructor<TermboxBindings> ) {
+	public primaryLanguageName = '';
+
+	public mounted() {
```

Comment:
"This hook is not called during server-side rendering."
https://vuejs.org/v2/api/#mounted